pkg/operation/cloudbotanist/gcpbotanist/infrastructure.go

```diff
@@ -155,6 +170,29 @@ func (b *GCPBotanist) destroyKubernetesFirewallRules(ctx context.Context) error
 	return nil
 }
 
+func (b *GCPBotanist) destroyKubernetesRoutes(ctx context.Context) error {
+	vpcName, err := b.getVpcName("Skipping explicit GCP route deletion because not all variables have been found in the Terraform state.")
```

Comment:
Instead of providing the log messages as argument, refactor it that way:

```golang
vpcName, err := b.getVpcName()
if err != nil {
  if terraformer.IsVariablesNotFoundError(err) {
    b.Logger.Infof("Skipping explicit GCP route deletion because not all variables have been found in the Terraform state.")
    return nil
  }
  return err
}
```

And in the `getVpcName()` function, just return the error if `t.GetStateOutputVariables` fails:

```golang
...
        stateVariables, err := t.GetStateOutputVariables(vpcNameID)
	if err != nil {
          return "", err
        }
...
```